src/shared/menu/helpers.ts

```diff
@@ -243,6 +244,13 @@ export function makeMenuButton(
         button.classList.add(...cssClasses);
     }
 
+    const title = typeof content === "string" ? content : content.title;
+    const helpText = typeof content === "string" ? null : content.helpText;
+
+    if (helpText) {
+        button.dataset.sTooltipHtmlTitle = `<p class="mb4">${title}</p><p class="fs-caption fc-black-600 m0">${helpText}</p>`;
```

Comment:
Could be a potential XSS here. Should use the html formatter helper instead.

Reply:
I thought that translations won't be considered user generated content. I guess better be extra safe nevertheless. 🙂